util/test/paratest.client

```diff
@@ -3,15 +3,16 @@
 # Client-side of the parallel testing script, paratest.server.
 # Used remotely by paratest.server to run start_test locally.
 #
-# Usage: paratest.client id chapeltestdir testdir chplenv futures-mode valgrind memleaks [compopts [execopts]]
+# Usage: paratest.client id chapeltestdir testdir chplenv futures-mode valgrind memleaks memleakslog [compopts [execopts]]
 #  
 #  id - used to create a file to synchronize with paratest.server
 #  chapeltestdir - root dir of Chapel test infrastructure
 #  testdir - directory to run start_test on
 #  chplenv - Chapel environment variables
 #  futures-mode - include .futures (0=no, 1=all, 2=futures only, 3=futures with skipifs)
 #  valgrind - run valgrind (0=no, 1=both compiler and executable, 2=executable only)
-#  memleaks - check for memleaks (0=no, 1=use -memleaks, 2=use -memleakslog)
```

Comment:
No strong preference, but would it be any cleaner to have the old behavior and have `memleaks=3` mean both? I'm always hesitant to add new flag passing between paratest server/client, but that's probably just paranoia and not based on anything.

Reply:
I can definitely see the reason for that paranoia.

However, I (also not strongly) prefer this way. Those numbers doesn't mean much to me, and even though I know that 2 would mean printing to a log file, I was never sure how to read the code when that was the case.